src/platforms/common/profiling/index.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: Set Up
+sidebar_order: 5000
+supported:
+  - android
+  - apple
+notSupported:
+  - unity
+  - dart
+  - flutter
+  - rust
+  - native
+  - php
+  - ruby
+  - javascript
+  - javascript.vue
+  - react-native
+  - dotnet
+  - python
+  - java
+  - java.spring-boot
+  - java.spring
+  - go
+  - javascript.cordova
+  - native.breakpad
+  - native.crashpad
+  - native.minidumps
+  - unreal
+  - native.wasm
+description: "Learn how to enable profiling in your app if it is not already set up."
+---
+
+<Note>
+
+Profiling is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. For access, contact us at [profiling@sentry.io](mailto:profiling@sentry.io).
+
+</Note>
+
+<Note>
+
+If you don't have access to Profiling please reach out to [profiling@sentry.io](mailto:profiling@sentry.io).
```

Comment:
Wondering why we need the second note at all actually. "Access" just means the ability for the customer to see profiling data in sentry.io, right? 

```suggestion
<Note>

Profiling is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. For access, contact us at [profiling@sentry.io](mailto:profiling@sentry.io). 
```

Reply:
Add yes. Good catch.